File: C++/Beecrowd/bee-2653.cpp
#include <bits/stdc++.h>
using namespace std;

int main(){
    set<string>djk;
    string str;
    while(cin >> str){
        djk.insert(str);
        str.clear();
    }
    cout << djk.size() << endl;

    return 0;
}

/* 2653 - Dijkstra
https://judge.beecrowd.com/en/problems/view/2653 */

File: Java/Beecrowd/bee2653.java
import java.io.IOException;
import java.util.*;

public class bee2653 {
 
    public static void main(String[] args) throws IOException {
        Scanner scan = new Scanner(System.in);
        Set<String>djk = new TreeSet<>();
        while(scan.hasNext()){
            String str = scan.nextLine();
            djk.add(str);
        }
        System.out.println(djk.size());
        scan.close();
    } 
}

/* 2653 - Dijkstra
https://judge.beecrowd.com/en/problems/view/2653 */
